Fixing Grammar (#11949)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/mac-jamfpro-policies.md

You need to make sure that all machines receiving Defender's package, also receive *all* configuration profiles listed above.

> [!WARNING]

> > JAMF supports Smart Computer Groups, that allow deploying e.g. configuration profiles or policies to all machines matching certain criteria evaluated dynamically.

Reviewer: ```suggestion
> JAMF supports Smart Computer Groups that allow deploying, such as configuration profiles or policies to all machines matching certain criteria evaluated dynamically.
```
